core/tests/sharing_tests.rs

```diff
@@ -929,9 +932,6 @@ fn move_write_shared_folder() {
     cores[0].sync(None).unwrap();
 
     cores[1].sync(None).unwrap();
-    cores[1]
-        .create_file("folder_link", roots[1].id, FileType::Link { target: folder.id })
-        .unwrap();
```

Comment:
keeping this caused a linkinsharedfolder error instead due to changed error precedence